porting/index.md

```diff
@@ -220,6 +220,17 @@ iSerial                 3 Mer Debug telnet on port 23 on usb0 192.168.2.15 - als
  [ 1234.123456] rndis_host 1-7:1.0 enp0s20f0u7: renamed from usb0
 ```
  * In this example shown above, `enp0s20f0u7` is the usb network device name. Use this for the USBNETWORK below
+ * Check if usb network device have a MAC address assigned.
+```
+$ ip address show dev USBNETWORK
+6: USBNETWORK: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
+    link/ether 00:00:00:00:00:00 brd ff:ff:ff:ff:ff:ff
+```
+If it shows the `00:00:00:00:00:00` as shown above, you will have to manually assign the MAC address,
```

Comment:
If it shows the address `00:00:00:00:00:00` as shown above,
OR:
If it shows `00:00:00:00:00:00` as shown above